lib/import/colorectal/providers/oxford/oxford_handler_colorectal.rb

```diff
@@ -84,7 +94,7 @@ def assign_variantpathclass(genocolorectal, record)
             else
               varpath = VAR_PATH_CLASS_MAP[varpathclass]
             end
-            genocolorectal.add_variant_class(varpath)
+            genocolorectal.add_variant_class(varpath) if varpath.present?
```

Comment:
Was this supposed to be part of this ticket?

Reply:
No @lauramccluskey1 , but I noticed a lot of time in logs this string being logged " Bad variant class string given: ; cannot proces" unnecessary consuming time ..so fixed it too..

Reply:
Ah ok that makes sense